### ISSUE TITLE
Nightly GitHub Action failure

### ISSUE THREAD

**Issue by robert-scheck** [rpki-client/rpki-client-portable#40]
Our nightly GitHub Action for the container image [failed](https://github.com/rpki-client/rpki-client-container/runs/2158093017?check_suite_focus=true#step:9:512) with:

```
#33 20.04 + ./autogen.sh
#33 20.04 pulling upstream openbsd source
#33 20.44 Already on 'master'
#33 20.44 Your branch is up to date with 'origin/master'.
#33 21.64 Already up to date.
#33 21.64 copying tal
#33 21.64 copying includes
#33 21.74 Copying as.c
#33 21.74 Copying cert.c
#33 21.75 Copying cms.c
#33 21.76 Copying crl.c
#33 21.76 Copying extern.h
#33 21.78 Copying gbr.c
#33 21.78 Copying http.c
#33 21.78 Copying io.c
#33 21.79 Copying ip.c
#33 21.79 Copying log.c
#33 21.80 Copying main.c
#33 21.80 Copying mft.c
#33 21.80 Copying mkdir.c
#33 21.82 Copying output-bgpd.c
#33 21.83 Copying output-bird.c
#33 21.83 Copying output-csv.c
#33 21.83 Copying output-json.c
#33 21.83 Copying output.c
#33 21.84 Copying parser.c
#33 21.89 Copying roa.c
#33 21.89 Copying rpki-client.8
#33 21.89 Copying rsync.c
#33 21.91 Copying tal.c
#33 21.91 Copying validate.c
#33 21.92 Copying x509.c
#33 21.92 Applying patch patches/0001-Allow-overriding-default-user-and-file-locations.patch
#33 21.92 1 out of 1 hunk FAILED -- saving rejects to file extern.h.rej
#33 ...
```

**Comment by cjeker** [comment]
This should be fixed by 12baf3c4b118448c5cad5cb59e7f9edf852e019b

**Comment by robert-scheck** [comment]
Yes, confirming [successful run](https://github.com/rpki-client/rpki-client-container/actions/runs/678279273) with commit 0fd7b2b4d7f00d0b730d3fe640827d1e475391da - thank you.
